botfront/imports/api/graphql/botResponses/resolvers/nlgResolver.js

```diff
@@ -1,6 +1,7 @@
 import { safeLoad } from 'js-yaml';
```

Comment:
Are those change still necessary? I thought that all the logic was now handled in Rasa